components/bl-modal-component/styles/index.less

```diff
@@ -0,0 +1,38 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
```

Comment:
This line could be deleted from our components.

Do you agree @Valodya?

Reply:
yes, I agree